.factory/automation.yml

```diff
@@ -24,32 +24,25 @@ config:
 build:
   correctness:
     build:
-      image: vaticle-ubuntu-20.04
+      image: vaticle-ubuntu-22.04
       command: |
         bazel build //...
         bazel run @vaticle_dependencies//tool/checkstyle:test-coverage
         bazel test $(bazel query 'kind(checkstyle_test, //...)') --test_output=errors
     test-unit:
-      image: vaticle-ubuntu-20.04
+      image: vaticle-ubuntu-22.04
       command: |
         bazel test $(bazel query 'kind(kt_jvm_test, //...) except kind(kt_jvm_test, //test/...)') --test_output=errors
     test-integration:
-      image: vaticle-ubuntu-20.04
+      image: vaticle-ubuntu-22.04
       command: |
         bazel build //test/integration/...
         bazel test $(bazel query '//test/integration/... except kind(checkstyle_test, //test/integration/...)') --jobs=1 --spawn_strategy=local --test_output=errors
 
 release:
-  #  filter:
-  #    owner: vaticle
-  #    branch: master
-  #  validation:
-  #    validate-dependencies:
-  #      image: vaticle-ubuntu-20.04
-  #      command: bazel test //:release-validate-nodejs-deps --test_output=streamed
```

Comment:
I think we should have `dependency-analysis` in its place (see `typedb`, etc., but maybe leave out `build-analysis` (also from `typedb`) because I don't know if it works in Kotlin)

Reply:
Done!